ci/licenses_golden/licenses_flutter

```diff
@@ -176,6 +176,8 @@ FILE: ../../../flutter/fml/dart/dart_converter.h
 FILE: ../../../flutter/fml/delayed_task.cc
 FILE: ../../../flutter/fml/delayed_task.h
 FILE: ../../../flutter/fml/eintr_wrapper.h
+FILE: ../../../flutter/fml/endianness.h
```

Comment:
Please add an empty `endianness.cc` that only include `endianness.h`. That is a good way to check that the header can always be included cleanly.

Reply:
Done.